webpack.config.js

```diff
@@ -0,0 +1,35 @@
+var path = require('path')
+// var nodeExternals = require('webpack-node-externals')
+
+module.exports = {
+  mode: process.env.NODE_ENV,
+  entry: './hyper-readings.es.js',
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    filename: 'hyper-readings.js',
+    library: 'HyperReadings',
+    libraryTarget: 'umd',
+    umdNamedDefine: true,
+    globalObject: `typeof self !== 'undefined' ? self : this`
+  },
+  externals: [
+    'fs',
+    'crypto'
+    // nodeExternals()
```

Comment:
currently we are building with all node dependencies bundled in. This is big, and there may be issues with parse5 caused by this. For now it is a convenience for bundling in hyper-reader. Will most likely fix this later.